views/welcome.ejs

```diff
@@ -34,5 +67,22 @@
 
     <p>We collect anonymous analytics using Simple Analytics. View live: <a href="#">simpleanalytics.com/learndatabases.dev</a></p>
 
+      <script>
+  const postgres = document.querySelector('.postgres')
+  const mongodb = document.querySelector('.mongodb')
+  const neo4j = document.querySelector('.neo4j')
+
+  postgres.addEventListener('click', () => {
```

Comment:
I just tested this page. These buttons are not valid because....
1. If users are not logged in, nothing happens because they will be redirected back to `"/"`
2. If users are logged in, they can't see these buttons because theses buttons won't be rendered.
    * Right now, `postgres` page is rendered when the user is logged in